edv/README.md

```diff
@@ -0,0 +1,51 @@
+# daos_scaled_testing/edv
+This directory contains DAOS setup and application execution scripts for the Intel cluster Endeavour.
+
+## Usage
+All scripts expect that the RUNDIR variable is set before any script is executed and that RUNDIR is the path to the client [directory]( https://github.com/daos-stack/daos_scaled_testing/tree/master/edv/test_scripts/client). For example, if the client directory is located at /panfs/users/janunez/, then we would set:
+export RUNDIR=/panfs/users/janunez/client
```

Comment:
You can probably replace janunez as <username>.

Reply:
Recommend formatting code
```suggestion
`export RUNDIR=/panfs/users/janunez/client`
```